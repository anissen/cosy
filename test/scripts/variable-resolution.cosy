var a = 'global'
{
  fn show_a() {
    print a
  }

  show_a()
  var a = 'block'
  print a
  show_a()
}
