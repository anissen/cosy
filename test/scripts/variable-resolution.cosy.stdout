â–  test/scripts/variable-resolution.cosy, line 8:
 6 | 
 7 |   show_a()
 8 |   var a = 'block'
           ^ Shadows existing variable.
 9 |   print a
10 |   show_a()

